bot/plugins/instance.py

```diff
@@ -210,6 +209,46 @@ async def on_component_interaction(event: hikari.InteractionCreateEvent) -> None
     await inst.execute()
 
 
+def create_input_args_modal(
+    event: hikari.InteractionCreateEvent,
+    msg_instance: Instance,
+):
+    stdin_modal_row = event.app.rest.build_modal_action_row()
+    stdin_modal_row.add_text_input(
+        ModalID.STDIN,
+        "Set STDIN",
+        value=msg_instance.stdin or hikari.UNDEFINED,
+        required=False,
+        style=hikari.TextInputStyle.PARAGRAPH,
+        max_length=1024,
+        placeholder="Standard Input",
+    )
+
+    comptime_modal_row = event.app.rest.build_modal_action_row()
+    comptime_modal_row.add_text_input(
+        ModalID.COMPTIME,
+        "Comptime Args",
+        value=msg_instance.comptime_args or hikari.UNDEFINED,
+        required=False,
+        style=hikari.TextInputStyle.PARAGRAPH,
+        max_length=255,
+        placeholder="Command Line Parameters - 1 per line",
+    )
+
+    runtime_modal_row = event.app.rest.build_modal_action_row()
+    runtime_modal_row.add_text_input(
+        ModalID.RUNTIME,
+        "Runtime Args",
+        value=msg_instance.runtime_args or hikari.UNDEFINED,
+        required=False,
+        style=hikari.TextInputStyle.PARAGRAPH,
+        max_length=255,
+        placeholder="Command Line Parameters - 1 per line",
```

Comment:
Let's use a different placeholder here than for compile args

Reply:
Any suggestions?

Reply:
Runtime Args - 1 per line